### ISSUE TITLE
Suggestion: Granite & Diorite Alloys

### ISSUE THREAD

**Issue by herbthyme** [MythrilBagels/Create-Alloyed#109]
There's already Andesite Alloy, so why not some alloys for granite & diorite? Mainly with decorative applications, their colours offer a nice bit of variety, and it could be expanded into some practical uses too. Still fits with the theme of Create, and can be integrated with other addons as this mod already is, such as Create Deco. Recipes could be similar to the existing Andesite Alloy recipe, using 2 of the block in question and 2 nuggets of a specific resource, perhaps gold nuggets for one, or it could stick to using iron nuggets for consistency.

**Comment by MythrilBagels** [comment]
You said they could have practical uses, yet didn't share any. I personally don't see how diorite or granite alloys are necessary at all.

**Comment by herbthyme** [comment]
Fair enough. I had intended to leave it open for interpretation, but coming back to this months later after forgetting I even suggested anything, I have no clue what my ideas were.

Best guess, just an alternative to andesite for stylistic usage, ie. andesite funnels & other contraptions. Though I can imagine that could easily mess up the regular progression system, unless they had an unnecessarily late-game recipe. Would be more in-line with another mod.
